MapboxNavigation/NavigationOptions.swift

```diff
@@ -48,12 +48,5 @@ open class NavigationOptions: NSObject, NavigationCustomizable {
         self.voiceController = voiceController
         self.bottomBanner = bottomBanner
     }
-    
-    /**
-    Convienence factory-method for convenient bridging to OBJ-C.
-     */
-    class func navigationOptions() -> Self {
```

Comment:
this is not exposed to Obj-C and it's not being used

Reply:
It should be exposed to Objective-C: https://github.com/mapbox/mapbox-navigation-ios/pull/1951#discussion_r252906378.

Reply:
Fixed in https://github.com/mapbox/mapbox-navigation-ios/pull/2013/commits/4a781a7482fb8d349dc8a9cbc46e472617d5b1c2